### PR TITLE
(maint) modsync update - master

### PR DESCRIPTION
**Pull request by glennsarti**
The PR contains the updates from the new modsync config for windows;

- LICENSE, NOTICE, CONTRIBUTING, MAINTAINERS updates (where applicable)
- Gemfile, Appveyor, Travis, gitignore cleanup

https://github.com/puppetlabs/modulesync_configs/pull/66
https://github.com/puppetlabs/modulesync_configs/pull/63
https://github.com/puppetlabs/modulesync_configs/pull/62
https://github.com/puppetlabs/modulesync_configs/pull/61

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@
   extras:
     - rvm: 1.9.3
       env: PUPPET_GEM_VERSION="3.0.0"
+  # FUTURE_PARSER testing is expected to fail and is disabled pending FM-5038
+  allow_failures:
+    - rvm: 2.0.0
+      env: PUPPET_GEM_VERSION="~> 3.0" FUTURE_PARSER="yes"
```
Comment (ferventcoder) [inline .sync.yml]: I'd almost prefer removing it entirely.

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Puppet Module - puppetlabs-chocolatey
+
+Copyright 2016 Puppet, Inc.
```
Comment (ferventcoder) [inline NOTICE]: This isn't going to fly.

Comment (glennsarti) [inline NOTICE]: Ok...what would you suggest?

Comment (ferventcoder) [inline NOTICE]:
~~~md
Copyright 2012 - 2014 Rich Siegel, Paul Morgan
Copyright 2014 - 2016 RealDimensions Software, LLC & original authors/contributors from https://github.com/chocolatey/puppet-chocolatey
Copyright 2016 Puppet, Inc
~~~

Comment (glennsarti) [inline NOTICE]: Updated

**Pull request merged**